nvflare/app_opt/xgboost/histogram_based_v2/defs.py

```diff
@@ -86,6 +86,7 @@ class Constant:
     PARAM_KEY_REPLY = "xgb.reply"
     PARAM_KEY_REQUEST = "xgb.request"
     PARAM_KEY_EVENT = "xgb.event"
+    PARAM_KEY_TRAINING_MODE = "xgb.training_mode"
```

Comment:
make sure we document different options for training mode

Reply:
The modes are described in detail in documentation.